### PR TITLE
Jp/refactoring new item8 test class

### PR DESCRIPTION
**Pull request by Yaroslav-R-P**
https://trello.com/c/VhkPFzOT/244-rf-refactoring-newitem8test-class 

Refactoring test NewItem8Test Class

1. Added empty lines where it was expected;
2. The itemName variable was made final;
3. Removed extra empty lines at the end of the test;
4. Rewrote all locators, got rid of text search. The search for linkText was also excluded.

### CODE REVIEW

**Comment by Yaroslav-R-P** [comment]
[RF _Refactoring NewItem8Test Class](https://trello.com/c/VhkPFzOT/244-rf-refactoring-newitem8test-class)

**Review by dope2young — APPROVED** [review]

File: src/test/java/school/redrover/NewItem8Test.java [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
-        getDriver().findElement(By.xpath("//button[text()='OK']")).click();
-
-        Assert.assertTrue(getDriver().findElement(By.linkText(itemName)).isDisplayed());
 
+        final String itemName = "My first Multibranch Pipeline";
 
+        getDriver().findElement(By.className("task")).click();
+        getDriver().findElement(By.className("jenkins-input")).sendKeys(itemName);
+        getDriver().findElement(By.xpath("(//li[@tabindex='0'])[5]")).click();
+        getDriver().findElement(By.xpath("//button[@type='submit']")).click();
```
Comment (AllaBird) [inline src/test/java/school/redrover/NewItem8Test.java:18]: Перед Assert лучше поставить пустую строку

File: src/test/java/school/redrover/NewItem8Test.java [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
+
     @Test
     public void testCreateItem() {
-        String itemName = "My first Multibranch Pipeline";
-        getDriver().findElement(By.linkText("New Item")).click();
-        getDriver().findElement(By.xpath("//input[@name='name']")).sendKeys(itemName);
-        getDriver().findElement(By.xpath("//span[text()='Multibranch Pipeline']")).click();
-        getDriver().findElement(By.xpath("//button[text()='OK']")).click();
-
-        Assert.assertTrue(getDriver().findElement(By.linkText(itemName)).isDisplayed());
```
Comment (AllaBird) [inline src/test/java/school/redrover/NewItem8Test.java:12]: здесь можно убрать пустую

**Review by AllaBird — APPROVED** [review]

File: src/test/java/school/redrover/NewItem8Test.java (hunk 1 above)
Comment (Yaroslav-R-P) [reply src/test/java/school/redrover/NewItem8Test.java:17]: Исправил, спасибо!

**Review by SavinovMA — APPROVED** [review]

**Pull request merged**